modules/lassoBidAdapter.js

```diff
@@ -44,6 +44,8 @@ export const spec = {
         sizes,
         aimXR,
         uid: '$UID',
+        npi: bidRequest.params.npi || '',
```

Comment:
Why not read the entire bidRequest.user.ext.data object so it isn't so clear your code is gathering pii?